src/cmd/pkg/ruleevaluations/list_test.go

```diff
@@ -60,13 +60,11 @@ func TestListRuleEvaluations(t *testing.T) {
 			},
 			expected: `rule_slug: slug-one
 rule_type: MONITOR
-detected_at: "1970-01-01T00:00:00.000Z"
```

Comment:
is this change related?

Reply:
discussed offline, and addressed by pinning strfmt in go.mod. Bug is described in a comment in go.mod, summarizing here:
```
// Required to avoid picking up https://github.com/go-openapi/strfmt/commit/b36a4901fcb51dbf4300a5a6b60b4335440aa208
// which causes strfmt.DateTime values to be incorrectly omitted from yaml output due to a bug in the yaml library
// where the IsZero method on pointers that implement the IsZeroer interface is not invoked.
```